browser/ui/webui/ai_chat/ai_chat_ui_page_handler.cc

```diff
@@ -147,6 +147,15 @@ void AIChatUIPageHandler::GetSiteInfo(GetSiteInfoCallback callback) {
                                                 : nullptr);
 }
 
+void AIChatUIPageHandler::OpenBraveLeoSettings() {
+  if (active_chat_tab_helper_) {
+    active_chat_tab_helper_->web_contents()->OpenURL(
+        {GURL("brave://settings/leo-assistant"), content::Referrer(),
```

Comment:
Could this be a constant in webui_url_constants?

Reply:
yeah but it's not going to be used by settings page anyway, so I think it's ok like this here since it's only used once